### PR TITLE
Change watermark settings

### PR DESCRIPTION
**Pull request by jackra1n**
Forgot the most important thing 😅

### CODE REVIEW

File: prepare-elasticsearch.sh [hunk 1]
```diff
@@ -69,6 +69,14 @@ echo "" >> $ES_FOLDER/config/jvm.options
 echo "-Xms1G" >> $ES_FOLDER/config/jvm.options
 echo "-Xmx1G" >> $ES_FOLDER/config/jvm.options
 
+# Add options to elasticsearch.yml
+echo "Adding options to elasticsearch.yml"
+echo "" >> $ES_FOLDER/config/elasticsearch.yml
+echo "cluster.routing.allocation.disk.threshold_enabled: true" >> $ES_FOLDER/config/elasticsearch.yml
```
Comment (alexsuter) [inline prepare-elasticsearch.sh:75]: can make only one echo and a multi line string?

File: prepare-elasticsearch.sh [hunk 2]
```diff
@@ -69,6 +69,14 @@ echo "" >> $ES_FOLDER/config/jvm.options
 echo "-Xms1G" >> $ES_FOLDER/config/jvm.options
 echo "-Xmx1G" >> $ES_FOLDER/config/jvm.options
 
+# Add options to elasticsearch.yml
+echo "Adding options to elasticsearch.yml"
+echo "" >> $ES_FOLDER/config/elasticsearch.yml
```
Comment (alexsuter) [inline prepare-elasticsearch.sh:74]: elasticsearch.yml should already exist? do you make it empty with this?

Comment (jackra1n) [reply prepare-elasticsearch.sh:74]: `echo "test" > myfile` overwrites a file
`echo "test" >> myfile` appends to a file

File: prepare-elasticsearch.sh [hunk 3]
```diff
@@ -65,9 +65,19 @@ cp log4j2.properties $ES_FOLDER/config/
 
 # Add options to jvm.options
 echo "Adding options to jvm.options"
-echo "" >> $ES_FOLDER/config/jvm.options
-echo "-Xms1G" >> $ES_FOLDER/config/jvm.options
-echo "-Xmx1G" >> $ES_FOLDER/config/jvm.options
+echo "
```
Comment (jackra1n) [inline prepare-elasticsearch.sh:68]: @alexsuter also made this multi line

**Review by alexsuter — APPROVED** [review]

**Pull request merged**